### PR TITLE
Fix non-sensical error message

### PR DESCRIPTION
**Pull request by The-Compiler**
Introduced in 12de92cd2b818906d342dbdfaf96999887bc9658 / #7698

### CODE REVIEW

**Review by RonnyPfannschmidt — APPROVED** [review]
good catch

should we add a follow-up issue to test the errors?

**Comment by The-Compiler** [comment]
Good idea, I added one.

File: testing/python/fixtures.py [hunk 1]
```diff
@@ -1092,6 +1092,26 @@ def test_1(arg):
         reprec = pytester.inline_run()
         reprec.assertoutcome(passed=2)
 
+    @pytest.mark.parametrize("name", ["path", "fspath", "module"])
+    def test_session_scoped_unavailable_attributes(self, pytester, name):
+        pytester.makepyfile(
+            f"""
+            import pytest
```
Comment (RonnyPfannschmidt) [inline testing/python/fixtures.py:1099]: i really wish we could do tests like this one as unittest

**Review by RonnyPfannschmidt — APPROVED** [review]
thanks, lovely turnaround

File: testing/python/fixtures.py (hunk 1 above)
Comment (The-Compiler) [reply testing/python/fixtures.py:1099]: IMHO that'd require fixtures to be importable/scoped in some way, rather than essentially being global state.

**Comment by The-Compiler** [comment]
Nevermind actually, this can be turned into an unit test without leaking the fixture to other tests by shuffling things around a bit and moving the test to class :sweat_smile:. WDYT @RonnyPfannschmidt?

**Comment by RonnyPfannschmidt** [comment]
@The-Compiler that is a most interesting solution, very creative and thanks!

we are still a few refactors away from being able to painlessly create the intermediate objects directly so this little hack is extremely effective

thanks!

**Review by bluetech — APPROVED** [review]
Whoops, thanks for fixing this!

**Pull request merged**